samples/tcp-echo/tcp-echo-services.yaml

```diff
@@ -22,6 +22,9 @@ spec:
   ports:
   - name: tcp
     port: 9000
+  - name: tcp-other
+    port: 9001
+  # Port 9002 is omitted intentionally for testing the pass through filter chain.
```

Comment:
is passthrough filter chain user facing?

Reply:
I think the feature should be user facing? Not sure if we document it somewhere on istio.io